tests/cli/commands/test_webserver_command.py

```diff
@@ -144,7 +144,7 @@ def test_cli_webserver_foreground_with_pid(self):
         proc.terminate()
         proc.wait()
 
-    @unittest.skipIf("TRAVIS" in os.environ and bool(os.environ["TRAVIS"]),
+    @unittest.skipIf("CI" in os.environ and bool(os.environ["CI"]),
```

Comment:
Is this still needed? It seems odd we don't have permission to do something in our own docker container.

(I guess this is a separate issue)

Reply:
I want to check it afterwards. I will make a separate PR for that.

Reply:
But yeah.. I will take the risk now :)